app.py

```diff
@@ -21,10 +21,10 @@ def predict():
     #predict = prediction[:, 1]

```

Comment:
GPT summary of [1fcf8a](https://github.com/SampatSuthar/ml-demo/blob/c51b3bc45df7b8f70fe55d6f0e781d925ddb587b/app.py#1fcf8aed8558b2618a97636c2f6d6c3587c2a6ee) - [1912d4](https://github.com/SampatSuthar/ml-demo/blob/27ac7dd34900e7bd89cba769d49ad7b986164e13/app.py#1912d43f95c066b874d8646e52a77052c42facc9):
* Removed the `openai` import 
* Commented out the `openai.api_key` line
* Changed the `model` variable from `text-davinci-003` to `davinci-003`